projects/ngneat/transloco/src/lib/transloco.service.ts

```diff
@@ -223,6 +223,36 @@ export class TranslocoService {
     }
   }
 
+  /**
+   * Returns the language code name from the browser, e.g. "de"
+   *
+   * @example
+   *
+   * service.getBrowserLang()
+   */
+  getBrowserLang() {
```

Comment:
It's great that `TranslocoService` provides a helper method: `getBrowserLang`, but it should also provide a method similar to the one in ngx-translate: [getBrowserCultureLang](https://github.com/ngx-translate/core/blob/master/projects/ngx-translate/core/src/lib/translate.service.ts#L514) since the app might use the long version of the locale (ex: en-US)